api/v1/controllers/bots.js

```diff
@@ -60,9 +62,10 @@ module.exports = {
    */
   getBot(req, res, next) {
     const resultObj = { reqStartTime: req.timestamp };
+    const key = req.swagger.params.key.value;
 
     // try to get cached entry
-    redisCache.get(req.swagger.params.key.value, (cacheErr, reply) => {
+    redisCache.get(`${botsRoute}/${key}`, (cacheErr, reply) => {
```

Comment:
use req.url

Reply:
@iamigo switched to req.url in commit 558d3a283346aa988b7716674b64bdf52d9098ae